client/landing/gutenboarding/onboarding-block/design-selector/style.scss

```diff
@@ -121,6 +129,12 @@ $deisgn-selector-selection-space: 175px;
 	overflow: hidden;
 	padding: 2px;
 
+	&:hover,
+	&:focus {
+		border: 3px solid var( --color-neutral-dark );
+		padding: 0;
+	}
```

Comment:
We can borrow from the [existing starter page template selector styles](apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/styles/starter-page-templates-editor.scss) - the suggestion below works for page selector cards. Still need to figure out how to work around a specificity collision with `.is-elevated` to use the same style on the design selector cards.

```
	&:focus {
		box-shadow: 0 0 0 1px $white,
					0 0 0 3px $blue-medium-500;
		// Windows High Contrast mode will show this outline, but not the box-shadow.
		outline: 2px solid transparent;
	}

	&:hover {
		border: 3px solid var( --color-neutral-dark );
		padding: 0;
	}
```

Reply:
Thanks for the suggestion. Opened this issue to try this out: https://github.com/Automattic/wp-calypso/issues/38665